cv32e40s/tests/cfg/pmp.yaml

```diff
@@ -0,0 +1,9 @@
+name: pmp
+description: Default pmp configuration for CV32E40S simulations
+compile_flags: >
+   +define+PMP_ENABLE_64
+ovpsim: >
+    --override root/cpu/PMP_registers=64
+cflags: >
+plusargs: >
+cv_sw_march: rv32imc
```

Comment:
While not a result of the merge - perhaps we should enable the zb*-instructions for the compiler as well here?

Reply:
I think Kun's PR will fix that? https://github.com/openhwgroup/core-v-verif/pull/1361/files#diff-3754c84dacc70eb0061c4ff9a4d8c746d662c63032ae35af064dc6aaf6ebf997R11